doc/changelog/12-infrastructure-and-dependencies/20464-ci+win_native.rst

```diff
@@ -0,0 +1,11 @@
+- **Added:** Experimental support for native Windows builds.
+  Rocq can now build and run under a native Windows environment using
+  the new native Windows support in Opam 2.3. This setup is tested in
+  CI, running a large part of the test suite. Beware this support is
+  still experimental, and some problem may arise on Unix-specific
+  tools. Note that RocqIDE is still not supported (c.f. `#20631 <https://github.com/rocq-prover/rocq/issues/20631>`_)
+  (`#20464 <https://github.com/rocq-prover/rocq/pull/20464>`_,
+  fixes `#9199 <https://github.com/rocq-prover/rocq/issues/9199>`_ and
+  `#17216 <https://github.com/rocq-prover/rocq/issues/17216>`_, by
```

Comment:
I'm adding those to the PR OP so that they'll be auto closed

Reply:
(although, are they really closed or does opam's cygwin include a bash?)

Reply:
Yes, using opam exec exposes bash, I had an older mode where I didn't use opam exec, and actually things worked too, I get confused with this all the time, as the cygwin exposed by opam is not the same than the one used in Rocq platform, will amend.